src/mixins/java/org/spongepowered/common/mixin/api/minecraft/world/entity/EntityMixin_API.java

```diff
@@ -167,7 +172,7 @@ public boolean setTransform(final Transform transform) {
         }
         Preconditions.checkNotNull(transform, "The transform cannot be null!");
         final Vector3d position = transform.position();
-        this.shadow$setPos(position.x(), position.y(), position.z());
+        this.setPosition(transform.position());
```

Comment:
Do not make API calls, the below code shouldn't either IF there is a bridge method.